MIDI Editor/Mordi_Search selected notes in chord finder.lua

```diff
@@ -12,10 +24,12 @@
 -- Function for opening a URL
 function OpenURL(url)
   local OS = reaper.GetOS()
-  if OS == "OSX32" or OS == "OSX64" then
+  if OS == "OSX32" or OS == "OSX64" or OS == "macOS-arm64" then
     os.execute('open "" "' .. url .. '"')
```

Comment:
To fix the bug where the home folder is also opened in finder:
```suggestion
    os.execute('open "' .. url .. '"')
```